packages/label/src/Label.js

```diff
@@ -19,63 +15,21 @@ const labelColor = props => {
   return colors.CARBON;
 };
 
-const TextLabel = styled(Text)`
-  order: 1;
-  margin-bottom: 8px;
-  float: left;
-`;
+const Label = styled.label`
+  font-family: Lato, sans-serif;
+  display: block;
```

Comment:
By having the label as a block, it takes the whole available width.
It seems weird in the example, but I'm not sure if this is good or bad in real usage as the inputs would be larger 🤔.

![2020-06-24 09 35 50](https://user-images.githubusercontent.com/1459283/85558978-a9e70900-b5ff-11ea-80a8-6a6317e25ab1.gif)

Reply:
Makes sense... It's probably a better idea to let hoc-input handle how it should be rendered

Reply:
I'll fix the margin bottom on this pr https://github.com/CraveFood/farmblocks/pull/988